x/bank/client/cli_test/send_test.go

```diff
@@ -0,0 +1,79 @@
+package cli_test
```

Comment:
ditto move to `client/cli/` pkg

Reply:
As discussed above, this is causing cyclic dependency as multiple modules need helpers from modules like bank.